pkg/reconciler/route/resources/ingress.go

```diff
@@ -253,8 +264,22 @@ func makeBaseIngressPath(ns string, targets traffic.RevisionTargets) *netv1alpha
 		})
 	}
 
+	var timeoutDuration *metav1.Duration
+
+	if timeout != nil {
+		if timeout.Seconds() > float64(defaults.MaxRevisionTimeoutSeconds) {
+			var t = time.Duration(defaults.MaxRevisionTimeoutSeconds) * time.Second
+			timeout = &t
+		}
+
+		// TODO: How should defaults.RevisionTimeoutSeconds be worked into this?
```

Comment:
I think the `defaults.RevisionTimeoutSeconds` is handled by Revision defaulting already, so we probably don't need to worry about this here.

Reply:
removed.